public_html/wp-content/plugins/wordcamp-speaker-feedback/views/view-feedback.php

```diff
@@ -41,6 +44,51 @@
 <div class="speaker-feedback">
 	<h2><?php esc_html_e( 'Session Feedback', 'wordcamporg' ); ?></h2>
 
+	<?php if ( ! $is_session_speaker ) : ?>
+		<p class="speaker-feedback__notice">
+			<?php esc_html_e( 'Only feedback recipients can mark submissions as helpful.', 'wordcamporg' ); ?>
+		</p>
+	<?php endif; ?>
+
+	<?php foreach ( $session_speakers as $session_speaker ) :
+		if ( $is_session_speaker && get_current_user_id() !== $session_speaker ) {
+			continue;
+		}
+		$user = new WP_User( $session_speaker );
+		$notifications_opt_out = get_user_meta( $user->ID, SPEAKER_OPT_OUT_KEY, true );
+		?>
+		<div class="speaker-feedback__notifications <?php echo ( $notifications_opt_out ) ? 'is-disabled' : ''; ?>">
+			<span>
+				<?php if ( ! $is_session_speaker ) : ?>
+					<span class="sft-notifications-speaker-name">
+						<?php echo esc_html( $user->user_login ); ?>
+					</span>
+				<?php endif; ?>
+				<span class="sft-notifications-label-text">
+					<?php if ( $notifications_opt_out ) : ?>
+						<?php esc_html_e( 'Feedback notifications are disabled.', 'wordcamporg' ); ?>
+					<?php else : ?>
+						<?php esc_html_e( 'Feedback notifications are enabled.', 'wordcamporg' ); ?>
+					<?php endif; ?>
+				</span>
+			</span>
+			<label>
+				<input
+					type="checkbox"
+					data-user-id="<?php echo absint( $user->ID ); ?>"
+					<?php checked( $notifications_opt_out ); ?>
```

Comment:
This needs an `aria-describedby` pointing to the content in the visual label, otherwise it's not totally clear to a screen reader user what you're enabling.

Reply:
Fixed with 4fe2df0